cogs/csgo.py

```diff
@@ -254,16 +260,47 @@ async def pug(self, ctx: commands.Context, *args):
 
         team1_country = 'IE'
         team2_country = 'IE'
+
+        team1_flags = []
+        team2_flags = []
+
+        team1_flag_request = ''
+        team2_flag_request = ''
+
+        for player in team1_steamIDs:
+            team1_flag_request += SteamID(player).__str__() + ','
+        team1_flag_request = team1_flag_request[:-1]
+
+        for player in team2_steamIDs:
+            team2_flag_request += SteamID(player).__str__() + ','
+        team2_flag_request = team2_flag_request[:-1]
+
         session = aiohttp.ClientSession()
-        async with session.get(f'http://api.steampowered.com/ISteamUser/GetPlayerSummaries/v0002/'
+        async with session.get(f'https://api.steampowered.com/ISteamUser/GetPlayerSummaries/v2/'
                                f'?key={self.bot.steam_web_api_key}'
-                               f'&steamids={SteamID(team1_steamIDs[0]).as_64},{SteamID(team2_steamIDs[0]).as_64}') as resp:
-            captain_info = await resp.json()
-            if 'loccountrycode' in captain_info['response']['players'][0]:
-                team1_country = captain_info['response']['players'][0]['loccountrycode']
-            if 'loccountrycode' in captain_info['response']['players'][1]:
-                team2_country = captain_info['response']['players'][1]['loccountrycode']
-        await session.close()
+                               f'&steamids={team1_flag_request}') as resp:
+            player_info = await resp.json()
+            for player in player_info['response']['players']:
+                if 'loccountrycode' in player:
+                    team1_flags.append(player['loccountrycode'])
+            await session.close()
+
+        session = aiohttp.ClientSession()
+        async with session.get(f'https://api.steampowered.com/ISteamUser/GetPlayerSummaries/v2/'
+                               f'?key={self.bot.steam_web_api_key}'
+                               f'&steamids={team2_flag_request}') as resp:
+            player_info = await resp.json()
+            for player in player_info['response']['players']:
+                if 'loccountrycode' in player:
+                    team2_flags.append(player['loccountrycode'])
+            await session.close()
+
+        # TODO: Add check for EU/CIS flag
+        if len(team1_flags) > 0:
+            team1_country = max(k for k, v in Counter(team1_flags).items() if v > 1)
```

Comment:
```suggestion
team1_country, _ = Counter(team1_flags).most_common(1)[0]
```
Would something like this make more sense? I believe currently it would retrieve the string max of the country flags.

For example `max(k for k, v in Counter(['IE', 'IE', 'IE', 'UK', 'UK']).items() if v > 1)` would return `UK` instead of `IE`

Reply:
You're right that's my bad

Reply:
Fixed in latest pr